src/azul/chalice.py

```diff
@@ -201,6 +201,10 @@ def _security_headers_middleware(self, event, get_response):
         response.headers['Strict-Transport-Security'] = f'max-age={seconds}; includeSubDomains'
         response.headers['X-Content-Type-Options'] = 'nosniff'
         response.headers['X-Frame-Options'] = 'DENY'
+        view_name = self.routes[event.path][event.method].view_name
+        handler = self.handler_map[view_name]
+        cache_control = getattr(handler, 'cache_control', 'no-store')
+        response.headers['Cache-Control'] = cache_control
```

Comment:
```suggestion
        view_function = self.routes[event.path][event.method].view_function
        cache_control = getattr(view_function, 'cache_control', 'no-store')
        response.headers['Cache-Control'] = cache_control
```

"handler" has different meaning in Chalice.